src/renderer/services/state/index.js

```diff
@@ -0,0 +1,14 @@
+import { send } from 'services/ipc';
+import { SET_KEY } from 'common/events';
+
+/**
+ * Sends the password to main process.
+ *
+ * @param {string} key - The user password.
+ * @returns {Promise<void>} - The promise to await.
+ */
+export const setKey = async key => send(SET_KEY, key);
```

Comment:
You don't need to specify `async` if you don't use `await`. You can use `await` on anything.

Reply:
Resolved